docs/components/modeler/bpmn/link-events/link-events.md

```diff
@@ -0,0 +1,45 @@
+---
+id: link-events
+title: "Link events"
+description: "Link events are pairs of intermediate events, one throwing and one catching, that link the two together."
+---
+
+Link events are pairs of intermediate events, one throwing and one catching, that link the two together.
+
+Link events have no significance related to content, but facilitate the diagram-creation process and are a mechanism for connecting two sections of a process. Link events can be used as intermediate events only.
```

Comment:
"Link events have no significant related to content" -- can you clarify this for me?

Reply:
This part was taken directly from https://camunda.com/bpmn/reference/#events-link

It is intended to describe that link events don't mean anything in the context of the process. No tasks must be performed (orchestrated), and the process doesn't wait for some event to occur. 

These events only make the diagrams easier to understand and reason about because you can link one part of the process with another without needing a sequence flow (arrow) between the two parts.